src/Coordination/KeeperContext.cpp

```diff
@@ -37,26 +38,11 @@ void KeeperContext::initialize(const Poco::Util::AbstractConfiguration & config,
 {
     dispatcher = dispatcher_;
 
-    if (config.hasProperty("keeper_server.availability_zone"))
+    const auto keeper_az = PlacementInfo::PlacementInfo::instance().getAvailabilityZone();
+    if (!keeper_az.empty())
     {
-        auto keeper_az = config.getString("keeper_server.availability_zone.value", "");
```

Comment:
should this old config path still be supported as a fallback for compatibility (at least for some period of time)?

Reply:
I think no, I confirmed with @incfly that it is not used anywhere, and we did not document it.